scripts/run_llama.py

```diff
@@ -200,14 +210,27 @@ def timing_cuda(
 if args.compile != "no":
     dynamic = args.compile == "dynamic"
     fullgraph = args.compile == "fullgraph"
-    model.forward = torch.compile(model.forward, mode="reduce-overhead", fullgraph=fullgraph, dynamic=dynamic)
+    mode = "reduce-overhead" if not args.sweep else "max-autotune"
```

Comment:
why?

Reply:
To see how fast we can get with compile :) Happy to revert if you feel like it won't be the planned use case!